### ISSUE TITLE
Detect more MIME types or all files.

### ISSUE THREAD

**Issue by Mundball** [cloud-py-api/mediadc#169]
**Description**
Detect more MIME types or all files, only with hash you can catch all files.

**Request**
Please add support for more MIME types.

Thanks.

**Comment by Cryx73** [comment]
Would a simple search for doubled files (by the file name) also be possible?

**Comment by Mundball** [comment]
No, the purpose of the hash is that you don't have to check if the file is really duplicated.

A little examples:

2023092600001.jpg = Neat_Donkey_Pic.jpg = holiday_pic_1.jpg
cake_recipe.txt = Choclate_Cake.txt = Best_Cake_ever.txt

How you search doubles from name? With a hash you get unique "code" for each file, it is much easier to diff over a hash.
